virtcontainers/shim.go

```diff
@@ -210,7 +210,6 @@ func startShim(args []string, params ShimParams) (int, error) {
 		cmd.SysProcAttr.Setsid = true
 		// Set Controlling terminal to Ctty
 		cmd.SysProcAttr.Setctty = true
```

Comment:
Shouldnt we set that one to false then?

Reply:
Ack, updated!

Reply:
Hmmm. It seems that I actually we shouldn't remove the `Setctty = true` call.

Removing it results in:
```
DEBU[0003] ExitCode msg: "container create failed (no logs from conmon): eof" 
Error: container create failed (no logs from conmon): EOF
```

Reply:
Ah, no, I tested the wrong branch. :-)
Let me mark this one as resolved again.